graders/r/Dockerfile

```diff
@@ -74,7 +74,7 @@ RUN apt-get update \
         ttdo \
         unix \
         xts \
-    && echo "options(diffobj.brightness=\"dark\")" >> /etc/R/Rprofile.site
+    && echo "options(diffobj.brightness=\"dark\")" >>/etc/R/Rprofile.site
```

Comment:
Hot take: I don't like this 🙃

Reply:
> `-sr, --space-redirects Redirect operators will be followed by a space.`

I'll add this flag to the formatter.

Reply:
Added in the `dockerfmt 0.3.0` release.

Reply:
It's pretty common to see no space in cases like `1>&2` though. Maybe worth another exception.